scripts/workers/submission_worker.py

```diff
@@ -216,7 +216,6 @@ def extract_challenge_data(challenge, phases):
         challenge_module = importlib.import_module(CHALLENGE_IMPORT_STRING.format(challenge_id=challenge.id))
         EVALUATION_SCRIPTS[challenge.id] = challenge_module
     except:
-        logger.exception('Exception raised while creating Python module for challenge_id: %s' % (challenge.id))
```

Comment:
We should catch this error, no? We will get an error on sentry if we are not able to start the worker.

Reply:
I removed it since we can always check it in the worker logs.

Reply:
True but this can be really painful when you have a lot of challenges and you launch them using one script that loops all the challenges and launches workers for all the challenges. You will then have to manually do `docker ps` and see which all challenge id is missing from the running containers and then do `docker logs <container_id>`. We can avoid all that by just keep this line, no?

Reply:
I agree.